openedx/core/djangoapps/oauth_dispatch/management/commands/edx_clear_expired_tokens.py

```diff
@@ -90,6 +90,12 @@ def handle(self, *args, **options):
         refresh_expire_at = self.get_expiration_time(now)
 
         if options['refresh-tokens']:
+            # remove revoked RefreshTokens
+            revoked = RefreshToken.objects.filter(revoked__lt=refresh_expire_at).exclude(
```

Comment:
1. Do we know the expected number of records for Prod?
2. Looks good. Do we have a test that shows this is actually doing what we want?

Reply:
It's a slow query so I will probably not have an answer before merging.